docs/app/pages/components/components-sections/tables/filters/filters.component.html

```diff
@@ -58,6 +58,9 @@
     <tr uxd-api-property name="clearTooltip" type="string" defaultValue="Clear All">
         Defines the text to display on the clear button tooltip.
     </tr>
+    <tr uxd-api-property name="clearAll" type="TemplateRef || string">
+        Overwrite the clear all button with a string.
```

Comment:
In the code you are using this as a template, so this is a bit misleading. Something like "Template to customize the content of the Clear All button" would be better.

Reply:
We also usually only have one `|` between types, and it is also good to document the TemplateRef context, eg. `TemplateRef<void>` or `TemplateRef<string>`